jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2396,6 +2414,8 @@ CAUTION: The scheduled workflows feature is set to be deprecated. Using *schedul
 
 A workflow may have a `schedule` indicating it runs at a certain time, for example a nightly build that runs every day at 12am UTC:
 
+Line 4:
+
```

Comment:
```suggestion
```

I think this was perhaps committed by mistake?
@rosieyohannan

Reply:
SO what I really wanted to do was highlight line four in the config snippet but line highlighting isn't working yet unfortunately so I added it above

<img width="954" alt="Screenshot 2024-06-25 at 10 34 05" src="https://github.com/circleci/circleci-docs/assets/24589403/92e7d65a-0009-4611-9ad5-fb31aeb73c37">

Reply:
I have removed for now until we have a better way